doc/source/user_guide/io.rst

```diff
@@ -6383,4 +6386,4 @@ The files ``test.pkl.compress``, ``test.parquet`` and ``test.feather`` took the
     24009288 Oct 10 06:43 test_fixed.hdf
     24009288 Oct 10 06:43 test_fixed_compress.hdf
     24458940 Oct 10 06:44 test_table.hdf
-    24458940 Oct 10 06:44 test_table_compress.hdf
+    24458940 Oct 10 06:44 test_table_compress.hdf
```

Comment:
not sure what's happened with the line ending - can you run the `pre-commit` checks on the files you've modified?